### PR TITLE
Remove redundant includes

### PR DESCRIPTION
**Pull request by ShikharJ**

### CODE REVIEW

File: symengine/cwrapper.cpp [hunk 1]
```diff
@@ -1,4 +1,3 @@
-#include <cstdlib>
```
Comment (isuruf) [inline symengine/cwrapper.cpp]: Why?

Comment (ShikharJ) [inline symengine/cwrapper.cpp]: `CLion` showed that this import was unused, and none of the builds failed as well.

Comment (isuruf) [inline symengine/cwrapper.cpp]: Does that mean it's getting imported implicitly by another header or that it's not used at all?

Comment (ShikharJ) [inline symengine/cwrapper.cpp]: I believe it's not used at all. `cwrapper.h` already includes `stdlib.h`.

**Comment by srajangarg** [comment]
LGTM @isuruf ?

**Comment by ShikharJ** [comment]
Ping @isuruf.

**Comment by ShikharJ** [comment]
@isuruf @certik Any objections here?

**Comment by ShikharJ** [comment]
Ping @isuruf

**Comment by ShikharJ** [comment]
As suggested by @isuruf, these changes may not be redundant across platforms.

**Pull request closed without merging**